### DISCUSSION TITLE
How can I change the sidebar from the left to the right side?

### DISCUSSION THREAD

**Discussion by emailbin** [zim-desktop-wiki/zim-desktop-wiki#2311] [Q&A] [answered]
Is it possible to set the sidebar in Zim to appear on the right side?
![Screenshot 2023-02-03 150308](https://user-images.githubusercontent.com/124371173/216622802-65e496c8-611b-4a1f-8abc-bb10c8da801f.png)

**Comment by bruvellu** [comment, marked as answer]
`Edit` > `Preferences` > `Plugins` > `Page Index` > `Configure` > `Position in the window` > `Right Side Pane`

**Comment by emailbin** [comment]
German: `Bearbeiten` > `Einstellungen` > `Erweiterungen` > `Seitenverzeichnis` > `Konfigurieren` > `Position im Fenster` > `Rechter Fensterbereich`
